### PR TITLE
Re-tune DP Accel Profile

### PR DESCRIPTION
**Pull request by wer5lcy**
Acceration profile is tuned to fit the 8ZR-FXE engine. 
- The normal mode uses the whole ECO range, so that the engine won't roar.
- The eco mode uses 75% of the ECO range, providing best fuel effiency.
- The sport mode uses 80% of the full engine power. 

Deceleration profile is changed such that the car could stop at static vehicles before traffic lights. 
- In previous version, using both accel profile "ECO" and following profile "1.2s" could cause your car unable to stop.
- The default max deceleration is 1.2m/s^2 in stock openpilot, any value in the custom profile should be no less than 1.2 unless necessary.

### CODE REVIEW

**Comment by wer5lcy** [comment]
Using 0.8.9 crash.py causes black screen on my device. I've to replace it with DP 0.8.8 crash.py to successfully boot.

**Comment by rav4kumar** [comment]
Revert the crash.py changes. And it can be merged.

File: selfdrive/crash.py [hunk 1]
```diff
@@ -1,67 +1,21 @@
 """Install exception handler for process crash."""
-import os
```
Comment (rav4kumar) [inline selfdrive/crash.py:2]: Revert crash.py

Comment (wer5lcy) [reply selfdrive/crash.py:2]: I'll close this pull request and pushes another request only change longitudinal_planner.py.

**Pull request closed without merging**